main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"unicode/utf8"
+)
+
+func max(words []string) string {
+	maxLength := utf8.RuneCountInString(words[0])
```

Comment:
Are you sure that `words` contains element with 0 index? 
What happens if I pass empty slice to the function ?